data/data/bootstrap/files/usr/local/bin/bootkube.sh.template

```diff
@@ -17,17 +17,24 @@ OPENSHIFT_HYPERKUBE_IMAGE=$(podman run --rm {{.ReleaseImage}} image hyperkube)
 
 mkdir --parents ./{bootstrap-manifests,manifests}
 
+# convert the release image pull spec to an "absolute" form if a digest is available
```

Comment:
Maybe do a pull and then inspect [here](https://github.com/openshift/installer/pull/780/files#diff-ce82c1d8a44f7dfc41dfc024085ccfeeR3) so that all call of `<cvo+releaseimage> image <component>` are from single image...?

Reply:
We won’t repull on the ones above - once we’ve resolved once only an explicit pull would pull again

Reply:
So the current semantics at least guarantee they’re all correct